### PR TITLE
Feature/sortbox setup

### LINKED ISSUE #12: As a user, I can organize planets based on various parameters
Detailed User Story:
As a user
In order to learn more about planets
When I choose a different organizational option
Then I view the planets in a different order based on that option (size, temperature, etc.)

Further Information:
Create a SortBox Component with Radio buttons to update Planetarium order
When a SortBox button is pressed, planets should automatically change order to reflect that option
Story is completed when a Sort Box is present on main page and pressing buttons change the order of planets and updates planet with that information (image, name, mean temperature)

### PR DESCRIPTION
**Pull request by knees4bees**
# PR PlanetParty
**Connor A-L, Alex T & Katie B**

### What does this do?

- [ ] Fix
- [x] Feature
- [ ] Refactor
- [ ] Style
- [ ] Testing

### Are there any known bugs?

- [x] No
- [ ] Yes (if so please disclose in Notes for Reviewer & Next Iterations)

### Summary of the changes
Creates a `SortBox` component with radio buttons and an `updateSort` function to sort the planets by whichever radio button is selected. Also updates file structure to create a `components` directory to hold all components.

### What ticket(s) do the changes resolve?
Resolves #12

Please Link Issues to this PR as necessary.

### Notes for the Reviewer
On my branch, all planet images are of Earth, so it's not easy to tell that the sort has occurred. I have a `console.log` inside the `updateSort` function for that purpose. [ETA: after resolving merge conflicts, the planet names are moving around to show the order (but keep in mind it's using fake data, so there's weird behavior like Mercury showing up with the highest number of moons).]

### How to Test Changes?
Try clicking on the various radio buttons and confirm (probably in the console) that the planets have been sorted into a different order.

### Next Iterations
Still need to style the radio buttons the way we discussed.

### CODE REVIEW

File: src/components/App/App.tsx [hunk 1]
```diff
@@ -1,21 +1,33 @@
...
 class App extends React.Component<{}, AllData> {
   constructor(props: any) {
     super(props);
     this.state = {
-      allPlanets: planetData
+      allPlanets: planetData,
     };
   }
 
+  updateSort = (event: React.MouseEvent<HTMLInputElement>): void => {
```
Comment (alexthompson207) [inline src/components/App/App.tsx:20]: This method is awesome!!! Great work with this

File: src/components/SortBox/SortBox.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
```
Comment (alexthompson207) [inline src/components/SortBox/SortBox.tsx:1]: Great work with this component

**Review by alexthompson207 — COMMENTED** [review]
Great work with the sortPlanet method! Super clean and easy to read!

**Review by ConnorAndersonLarson — APPROVED** [review]
Looks great, awesome work getting this going!!!!

**Pull request merged**